kubernetes/core-job.yml

```
@@ -0,0 +1,17 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: core-job
+spec:
+  completions: 1
+  backoffLimit: 2
+  template:
+    spec:
+      containers:
+        - name: curl-container
+          image: appropriate/curl
+          args:
+            - /bin/sh
+            - -c
+            - curl -v http://core-app-svc:5000/health;
```

There can be one optimization. If you check the image [Dockerfile](https://github.com/appropriate/docker-curl/blob/master/latest/Dockerfile), you will see the command is already set and there is only need to specify arguments.

```suggestion
            - -v
            - http://core-app-svc:5000/health
```

---

Cool! It works, thank you for the hint